worldedit-core/src/main/java/com/sk89q/worldedit/world/registry/ItemRegistry.java

```diff
@@ -63,7 +63,10 @@ default String getName(ItemType itemType) {
      *
      * @param itemType the item
      * @return the material, or null if the material information is not known
+     * @deprecated ItemMaterial is deprecated with no alternative.
      */
+    @Deprecated(forRemoval = true)
     @Nullable
+    @SuppressWarnings("removal")
```

Comment:
Is this really needed?

Reply:
Yes, any reference to something deprecated for removal is now erroring the build due to producing a warning, even if the current block is deprecated too